pom.xml

```diff
@@ -116,12 +121,13 @@
 		</dependency>
 
 	</dependencies>
-	
+
 	<build>
     <plugins>
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
+        <version>2.6.7</version>
```

Comment:
Shouldn't this be driven by dependency management? If we need a higher version, I'd prefer to set it there.